README.rst

```diff
@@ -1,3 +1,15 @@
+.. image:: https://img.shields.io/gitter/room/translate/virtaal.svg?style=flat-square
+   :alt: Join the chat at https://gitter.im/translate/virtaal
```

Comment:
this gitter room does not exist.  You did not test this.
Please remove.

Also, please  use `git commit --amend` to modify your current commit on this pull request.  Do not create a new pull request about his change.  Fix this pull request.